nixio/exceptions/__init__.py

```diff
@@ -7,5 +7,5 @@
     "DuplicateName", "UninitializedEntity", "InvalidUnit",
     "InvalidAttrType", "InvalidEntity", "OutOfBounds",
     "IncompatibleDimensions", "InvalidFile",
-    "DuplicateColumnName", "UnsuportedLinkType"
```

Comment:
How did you find that one, impressive ;)

Reply:
I think the linter caught it :)